### DISCUSSION TITLE
Article Image preview in list Layout

### DISCUSSION THREAD

**Discussion by ahmed-khlifi** [dailydotdev/daily#1065] [Feature Requests]
I've been using daily dev for more than a year, and I Love it!! this a little request for the List layout.
The list layout is a great option when I open the extension in a vertical monitor, however, the articles image preview are not shown in this layout which make it a little bit boring since previews helps to catch attention 
It will be nice to add it in upcoming updates ! 
Thank You and keep up the good work 👌❤️
![image](https://github.com/dailydotdev/daily/assets/47308272/1243277f-e026-4315-ad21-fe8be20da5b5)

**Comment by idoshamun** [comment]
Hi @ahmed-khlifi,

Thank you for the kind words 🙏 
We're thinking of revamping this layout. Stay tuned!

**Comment by idoshamun** [comment]
There's a new layout now with image preview. So I'm going to close this one
